cle/backends/relocations/peamd64.py

```diff
@@ -0,0 +1,21 @@
+import logging
+from . import generic_pe
+
+l = logging.getLogger('cle.relocations.peamd64')
+
+arch = 'peAMD64'
```

Comment:
Yes, I was going to suggest that we move the base relocations in to `cle/backends/relocation.py` and then the ELF ones into `cle/backends/elf/relocation/` and the PE ones into `cle/backends/pe/relocation{.py, /}`